pkg/cli/demo_cluster.go

```diff
@@ -883,29 +893,26 @@ func generateCerts(certsDir string) (err error) {
 }
 
 func (c *transientCluster) getNetworkURLForServer(
-	serverIdx int, gen workload.Generator, includeAppName bool,
+	serverIdx int, includeAppName bool,
 ) (string, error) {
 	options := url.Values{}
 	if includeAppName {
 		options.Add("application_name", catconstants.ReportableAppNamePrefix+"cockroach demo")
 	}
 	sqlURL := url.URL{
-		Scheme: "postgres",
+		Scheme: "postgresql",
```

Comment:
would it be worth introducing a `const PostgreSQLScheme = "postgresql"` somewhere so that all these url builders can refer to the same thing?

(btw, i hope there aren't any drivers out there that only accept `postgres` and not `postgresql`. i haven't been able to find one yet)

Reply:
Done.